include/task.h

```diff
@@ -1597,6 +1597,27 @@ UBaseType_t uxTaskGetStackHighWaterMark( TaskHandle_t xTask ) PRIVILEGED_FUNCTIO
  */
 configSTACK_DEPTH_TYPE uxTaskGetStackHighWaterMark2( TaskHandle_t xTask ) PRIVILEGED_FUNCTION;
 
+/**
+ * task.h
+ * @code{c}
+ * uint8_t* pxTaskGetStackStart( TaskHandle_t xTask);
+ * @endcode
+ * 
+ * INCLUDE_pxTaskGetStackStart must be set to 1 in FreeRTOSConfig.h for
+ * this function to be available.
+ *
+ * Returns the start of the stack associated with xTask. That is,
+ * the highest stack memory address on architectures where the stack grows down
+ * from high memory, and the lowest memory address on architectures where the
+ * stack grows up from low memory.
+ *
+ * @param xTask Handle of the task associated with the stack returned.
+ * Set xTask to NULL to return the stack of the calling task.
+ *
+ * @return A pointer to the start of the stack.
+ */
+uint8_t* pxTaskGetStackStart( TaskHandle_t xTask) PRIVILEGED_FUNCTION;
```

Comment:
You can get this information using  vTaskGetInfo API - 
* https://github.com/FreeRTOS/FreeRTOS-Kernel/blob/main/include/task.h#L1041
* https://github.com/FreeRTOS/FreeRTOS-Kernel/blob/main/include/task.h#L163

Reply:
I agree. Removed it from the patchset.